gammapy/image/utils.py

```diff
@@ -121,11 +122,20 @@ def disk_correlate(image, radius, mode='constant'):
 
     Parameters
     ----------
-    TODO
+    image : ~numpy.ndarray
```

Comment:
This formatting is not OK ... it will not generate a Sphinx inter-link to the ndarray description in the numpy docs.
```
~numpy.ndarray
```
You have to add single backticks:
```
`~numpy.ndarray`
```